tools/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -738,6 +751,8 @@ private void NetworkAddressChanged(object sender, EventArgs evt)
             {
                 eventLog.WriteEntry("Failed to delete the route to the proxy after a network change.",
                                     EventLogEntryType.Error);
+                ResetRoutingOnNetworkError();
+                return;

```

Comment:
It looks like any other routes we've added won't be deleted if this fails - that's a change from the current behaviour (and is probably fine) but is it what you intended?

Reply:
Which routes are you referring to? `ResetRoutingOnNetworkError` calls `ResetRouting`, which should restore the system routes. Am I missing something?

Reply:
*click* Got it, thanks.